docs/_posts/2014-04-14-pagination-strategies-with-pouchdb.md

```diff
@@ -423,4 +423,4 @@ So congratulations, you're now a [Page Master]!  Try to use your newfound powers
   [collation-specification]: http://docs.couchdb.org/en/latest/couchapp/views/collation.html#collation-specification
   [query-api]: http://pouchdb.com/api.html#query_database
   [page master]: https://en.wikipedia.org/wiki/Pagemaster
-  [remove]: http://pouchdb.com/api.html#delete_document
\ No newline at end of file
+  [remove]: http://pouchdb.com/api.html#delete_document
```

Comment:
I can't see any difference here. This have been edited within GitHub web interface.